dataServices.js

```diff
@@ -0,0 +1,56 @@
+'use strict';
+
+const products = require('./src/data/products');
```

Comment:
Don't load it in advance, load it when productMap is null inside the getProducts function